src/index/rtx.rs

```diff
@@ -3,7 +3,7 @@ use super::*;
 pub(crate) struct Rtx<'a>(pub(crate) redb::ReadTransaction<'a>);
```

Comment:
Why do we have the this `Rtx` file and don't just do it directly in the `index.rs`?`